projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -328,10 +329,40 @@ export class IgxOverlayService implements OnDestroy {
             return null;
         }
 
+        let proxy = {
```

Comment:
add the `info` object here as well as all the methods you need like this:
```ts
let proxy = {
   ...your code...
   info: info,
   addOutsideClickListener: this.addOutsideClickListener,
   ... more functions here ...
}

This will allow you then to call methods like this:
```ts
this.addOutsideClickListener(this.info);
```